mk/spksrc.cross-rust-env.mk

```diff
@@ -0,0 +1,62 @@
+# Configuration for rust build
+# 
+
+RUST_TOOLCHAIN ?= stable
+
+RUST_TARGET =
+# map archs to rust targets
+ifeq ($(findstring $(ARCH), $(x64_ARCHS)),$(ARCH))
+RUST_TARGET = x86_64-unknown-linux-gnu
+endif
+ifeq ($(findstring $(ARCH), $(i686_ARCHS)),$(ARCH))
+RUST_TARGET = i686-unknown-linux-gnu
+endif
+ifeq ($(findstring $(ARCH), $(ARMv5_ARCHS)),$(ARCH))
+# may be not supported for cargo
+RUST_TARGET = armv5te-unknown-linux-gnueabi
+endif
+ifeq ($(findstring $(ARCH), $(ARMv7_ARCHS)),$(ARCH))
+RUST_TARGET = armv7-unknown-linux-gnueabihf
+endif
+ifeq ($(findstring $(ARCH), $(ARMv7L_ARCHS)),$(ARCH))
+RUST_TARGET = armv7-unknown-linux-gnueabi
```

Comment:
This should be using -gnueabihf according to https://static.rust-lang.org/rustup/rustup-init.sh